interfaces/PA-App/src/app/personal-components/personal-components.enum.ts

```diff
@@ -20,5 +20,5 @@ export const PersonalComponentsRemoved = [
   'select-appointment',
   'select-country',
   'store-credential',
-  'inclusion-status',
```

Comment:
Haha! I guess this got replaced by all "`handle-proof` to `inclusion-status`"-replacements? :D

Reply:
yes, see follow-up here https://dev.azure.com/redcrossnl/121%20Platform/_sprints/taskboard/121%20Development%20Team/121%20Platform/Sprint%2047?workitem=7230